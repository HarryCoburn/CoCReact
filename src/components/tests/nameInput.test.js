import NameInput from "../nameInput";
import React from "react";
import renderer from "react-test-renderer";

describe("NameInput Component", () => {
  it("renders correctly", () => {
    const component = renderer.create(<NameInput></NameInput>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
