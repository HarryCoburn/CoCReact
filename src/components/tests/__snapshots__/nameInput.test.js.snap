// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NameInput Component renders correctly 1`] = `
<input
  onChange={[Function]}
/>
`;
